### PR TITLE
[New Process] - Move EthFlow & TradeSummary code to cow-react/modules/swap

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

Moves `custom/components/swap/EthFlow` and `custom/components/swap/TradeSummary` to `cow-react/modules/swap/EthFlow` and `cow-react/modules/swap/TradeSummary`, respectively.

## Testing
THis just moves eth flow + trade summary components into the @cow folders.
Testing should just be to confirm the app works as normal (no added features)

### CODE REVIEW

**Comment by W3stside** [comment]
discussed with @shoom3301 will move these to their proper home inside `@cow/modules/swap` once #1209 is merged

**Comment by W3stside** [comment]
Hi @elena-zh could you kindly test this branch? Thank you!

File: src/cow-react/modules/swap/containers/RowReceivedAfterSlippage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import TradeGp from 'state/swap/TradeGp'
+import { computeSlippageAdjustedAmounts } from 'utils/prices'
+import { RowWithShowHelpersProps } from '@cow/modules/swap/pure/Row/typings'
+
+export interface RowReceivedAfterSlippageProps extends RowWithShowHelpersProps {
+  trade: TradeGp
+  allowedSlippage: Percent
+}
+
+export function RowReceivedAfterSlippage({ trade, allowedSlippage, showHelpers }: RowReceivedAfterSlippageProps) {
```
Comment (shoom3301) [inline src/cow-react/modules/swap/containers/RowReceivedAfterSlippage/index.tsx:17]: The component doesn't have any hooks-dependencies, suppose to be pure

**Review by shoom3301 — APPROVED** [review]

**Review by elena-zh — APPROVED** [review]
@W3stside , great job, looks good!

**Pull request merged**